label_sync/labels.md

```diff
@@ -335,5 +335,6 @@ larger set of contributors to apply/remove them.
 | <a id="language/no" href="#language/no">`language/no`</a> | Issues or PRs related to Norwegian language| anyone |  [label](https://git.k8s.io/test-infra/prow/plugins/label) |
 | <a id="language/pt" href="#language/pt">`language/pt`</a> | Issues or PRs related to Portuguese language| anyone |  [label](https://git.k8s.io/test-infra/prow/plugins/label) |
 | <a id="language/zh" href="#language/zh">`language/zh`</a> | Issues or PRs related to Chinese language <br><br> This was previously `language/cn`, | anyone |  [label](https://git.k8s.io/test-infra/prow/plugins/label) |
+| <a id="deprecated" href="#triage/deprecated">`docs/deprecated`</a> | Issues or PRs related to deprecated documentation | anyone |  [label](https://git.k8s.io/test-infra/prow/plugins/label) |
```

Comment:
Is the label `triage/deprecated` or `docs/deprecated`?

If it is the former, then run `bazel run //hack:update-labels` locally to fix the test failures and re-push.

Reply:
Is this supposed to only intended to be used in the website repo? If so, would `kind/deprecated` make more sense?

Reply:
@nikhita 👋 

> Is this supposed to only intended to be used in the website repo? 

As far as I know.

> If so, would kind/deprecated make more sense?

We intend to use the label for issue triage, but possibly? What are the important differences between `triage` and `kind`?